rskj-core/src/main/java/co/rsk/peg/federation/FederationMemberPubKeysComparator.java

```diff
@@ -0,0 +1,40 @@
+package co.rsk.peg.federation;
+
+import com.google.common.primitives.UnsignedBytes;
+
+import java.util.Comparator;
+
+public class FederationMemberPubKeysComparator implements Comparator<FederationMember> {
+    /**
+     * Compares federation members based on their underlying keys.
+     *
+     * The total ordering is defined such that, for any two members M1, M2,
+     * 1) M1 < M2 iff BTC_PUB_KEY(M1) <lex BTC_PUB_KEY(M2) OR
+     *              (BTC_PUB_KEY(M1) ==lex BTC_PUB_KEY(M2) AND
+     *               RSK_PUB_KEY(M1) <lex RSK_PUB_KEY(M2)) OR
+     *              (BTC_PUB_KEY(M1) ==lex BTC_PUB_KEY(M2) AND
+     *               RSK_PUB_KEY(M1) ==lex RSK_PUB_KEY(M2) AND
+     *               MST_PUB_KEY(M1) <lex MST_PUB_KEY(M2))
+     * 2) M1 == M2 iff BTC_PUB_KEY(M1) ==lex BTC_PUB_KEY(M2) AND
+     *                 RSK_PUB_KEY(M1) ==lex RSK_PUB_KEY(M2) AND
+     *                 MST_PUB_KEY(M1) ==lex MST_PUB_KEY(M2) AND
+     * 3) M1 > M2 otherwise
```

Comment:
```suggestion
     * 1) M1 < M2 if BTC_PUB_KEY(M1) <lex BTC_PUB_KEY(M2) OR
     *              (BTC_PUB_KEY(M1) ==lex BTC_PUB_KEY(M2) AND
     *               RSK_PUB_KEY(M1) <lex RSK_PUB_KEY(M2)) OR
     *              (BTC_PUB_KEY(M1) ==lex BTC_PUB_KEY(M2) AND
     *               RSK_PUB_KEY(M1) ==lex RSK_PUB_KEY(M2) AND
     *               MST_PUB_KEY(M1) <lex MST_PUB_KEY(M2))
     * 2) M1 == M2 if BTC_PUB_KEY(M1) ==lex BTC_PUB_KEY(M2) AND
     *                 RSK_PUB_KEY(M1) ==lex RSK_PUB_KEY(M2) AND
     *                 MST_PUB_KEY(M1) ==lex MST_PUB_KEY(M2) AND
     * 3) M1 > M2 otherwise
```

Does lex mean lexicographical order?